RCode/NEFI/Jags_Models/RandomYear.R

```diff
@@ -14,21 +14,17 @@ m[i] <- exp(mu[i])
 #### Process Model
 for(i in 2:N){
 mu[i]~dnorm(x[i],tau_add) #mu's here are on log scale
-#x[i] <- x[i-1] + beta[1] # Adding fixed intercept
-#x[i] <- x[i-1] + yr[year_no[i]] # Adding year as a random effect
-x[i] <- x[i-1] + beta[1] + yr[year_no[i]] # Adding year as a random effect + fixed effect of intercept
+x[i] <- x[i-1] + yr[year_no[i]] # Adding year as a random effect
 }
 
 #### Priors
 x[1] ~ dnorm(x_ic,tau_ic) 
 tau_add ~ dgamma(a_add,r_add)
 mu[1] ~ dgamma(1,1)
-beta[1] ~ dnorm(beta.m,beta.v)
 tau_yr ~ dgamma(0.01,0.01)
 
 #Loops through number of years and defines prior for each year 
-for(i in length(year_no)) {
+for(i in 1:8) {
```

Comment:
Are we always going to have 8 years? maybe make this more flexible by having `1:length(year_no)`

Reply:
Thanks! We actually want to number of unique years and I think length would do the whole vector, but I'll change it to do the number of unique years!